src/main/resources/logback.xml

```diff
@@ -15,17 +15,16 @@
     limitations under the License.
 -->
 <configuration>
-
-  <appender name="STDOUT"
-    class="ch.qos.logback.core.ConsoleAppender">
+  <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
     <encoder>
-      <pattern>%level %logger{36} - %msg%n</pattern>
```

Comment:
I think the logger name is actually rather useful (although primarily in debug scenarios)

Reply:
I don't suppose it's possible to change the pattern based on the log level? This is a consequence of trying to use the log for user output as well as debugging. The output was meant to look similar to say, Maven, which doesn't output this information.

I wonder if there's some smaller piece of information (class name?) that might give us the same insight that might be easier to read for a "normal" user.

Reply:
Having it in for debug level only might be a nice compromise :-) I think thats possible with two different appender-refs?